control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -2116,3 +2122,22 @@ func (r *reconciler) reconcileImageContentPolicyType(ctx context.Context, hcp *h
 	}
 	return nil
 }
+
+// allLoadBalancersRemoved checks any service of type corev1.ServiceTypeLoadBalancer exists.
+// If any one service of type corev1.ServiceTypeLoadBalancer exists, will return false or else will return true.
+func allLoadBalancersRemoved(ctx context.Context, c client.Client) (bool, error) {
+	log := ctrl.LoggerFrom(ctx)
+	list := &corev1.ServiceList{}
+	if err := c.List(ctx, list); err != nil {
+		return false, fmt.Errorf("cannot list %T: %w", list, err)
+	}
+
+	for _, svc := range list.Items {
+		if svc.Spec.Type == corev1.ServiceTypeLoadBalancer {
+			log.Info("load balancer services are still exists")
```

Comment:
I think we can improve this log message, e.g?
```suggestion
			log.Info("Waiting on service %s of type LoadBalancer to be deleted", svc.Name)
```

otherwise lgtm

Reply:
Addressed